docs/source/general/how_to_extend_django_freeradius.rst

```diff
@@ -0,0 +1,114 @@
+    DJANGO_FREERADIUS_RADIUSPOSTAUTHENTICATION_MODEL = "my_radius_app.RadiusPostAuth"
+
+    # where my_radius_app is name of your app extending django_freeradius
+
+
+Extend models
+-------------
+
+Apart from extending implemented models, ``django_freeradius`` also provides flexibility to extend abstract class models from `django-freeradius.base.models`.
+
+Example:
+
+.. code-block:: python
+
+    # In my_radius_app/models.py
+
+    from django.db import models
+    from django_freeradius.base.models import AbstractRadiusCheck
+    
+    class RadiusCheck(AbstractRadiusCheck):
+        # modify/extend the default behavour here
+        custom_field = models.TextField()
+
+
+Extend admin
+------------
+
+Similar to models, abstract admin classes from ``django_freeradius.base.admin`` can also be extended to avoid duplicate code.
+
+.. code-block:: python
+
+    # In my_radius_app/admin.py
+    
+    from django.contrib import admin 
+    from .models import RadiusCheck
+    from django_freeradius.base.admin import AbstractRadiusAccountingAdmin
+
+    class RadiusCheckAdmin(AbstractRadiusCheckAdmin):
+        model = RadiusCheck
+        # modify/extend default behaviour here
+        fields = AbstractRadiusCheckAdmin.fields + ['custom_field']
+        list_display = AbstractRadiusCheckAdmin.list_display + ['custom_field']
+
+    admin.site.register(RadiusCheck,RadiusCheckAdmin)
+
+Extend AppConfig
+----------------
+
+You can also extend AppConfig class from ``django_freeradius.apps.DjangoFreeradiusConfig`` and provide support for your signals and hooks. 
+
+.. code-block:: python
+
+    # In my_radius_app/apps.py
+    
+    from django.conf import settings
+    from django_freeradius.apps import DjangoFreeradiusConfig
+    from django.core.exceptions import ImproperlyConfigured
+
+    API_TOKEN = settings.DJANGO_FREERADIUS_API_TOKEN
+
+    class MyRadiusAppConfig(DjangoFreeradiusConfig):
+        name = 'my_radius_app'
+
+        # Overiding DjangoFreeradiusConfig.check_settings
+        def check_settings(self):
+            if API_TOKEN and len(API_TOKEN) < 20 or not API_TOKEN:
+                raise ImproperlyConfigured('Security error: DJANGO_FREERADIUS_API_TOKEN '
+                                        'is either not set or is less than 20 characters.')   
+
+
+Extend API views
+----------------
+
+You can also extend API views from ``django_freeradius.api.views`` to your suit your models.
+
+.. code-block:: python
+
+    # In my_radius_app/api/views.py
+
+    from django_freeradius.api.views import AuthorizeView,AuthorizeView
+
+    class MyRadiusTokenAuthentication(TokenAuthentication):
+        # modify/extend default behaviour here
+        pass
+
+    class MyRadiusAuthorizeView(AuthorizeView):
+        # use your modified authentication class
+        authentication_classes = (SampleTokenAuthentication,)
+
+    authorize = SampleAuthorizeView.as_view()
+
```

Comment:
Add some links to openwisp-radius, explaining that for a real world example they should take a look here:

https://github.com/openwisp/openwisp-radius/blob/master/openwisp_radius/api